### PR TITLE
Adding refinitiv-data library recipe

### PR DESCRIPTION
**Pull request by Mhoian**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Mhoian** [comment]
@conda-forge/help-python The recipe has passed all checks and is redady for your review. Thanks

File: recipes/refinitiv-data/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (dopplershift) [inline recipes/refinitiv-data/LICENSE:1]: The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just need to remove the unneeded license file. Also, I need confirmation from @PFaurel that they are willing to be listed as a maintainer for this recipe.

**Comment by PFaurel** [comment]
> Just need to remove the unneeded license file. Also, I need confirmation from @PFaurel that they are willing to be listed as a maintainer for this recipe.

I confirm

File: recipes/refinitiv-data/LICENSE (hunk 1 above)
Comment (Mhoian) [reply recipes/refinitiv-data/LICENSE]: License removed and got the confirmation from @PFaurel

File: recipes/refinitiv-data/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{% set name = "refinitiv-data" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/refinitiv-data-{{ version }}.tar.gz
+  sha256: c4ad0698a9ed2bdf11af836faeb7ea797e72edc048df043301307f1e1b980d88
```
Comment (dopplershift) [inline recipes/refinitiv-data/meta.yaml:10]: Apparently the license was only included with 1.1.0:
```suggestion
{% set version = "1.1.0" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/refinitiv-data-{{ version }}.tar.gz
  sha256: 471a0decd1e16d70c25084f33439a0038ba65a99c15e571fd55d03c766aceb41
```

Comment (Mhoian) [reply recipes/refinitiv-data/meta.yaml]: done

**Review by dopplershift — APPROVED** [review]
Pending CI.

**Pull request merged**